### ISSUE TITLE
Failed to load plugin - invalid signature

### ISSUE THREAD

**Issue by hmitsch** [grafana/google-sheets-datasource#97]
We are trying to run `grafana-googlesheets-datasource` in Grafana 7.0.0-beta3 (`grafana/grafana:7.0.0-beta3` docker container).

Unfortunately, an invalid signature is reported:

```
"Failed to load plugin" logger=plugins error="plugin \"grafana-googlesheets-datasource\" has an invalid signature" pluginPath=/var/lib/grafana/plugins/grafana-googlesheets-datasource/dist
```

Everything worked fine on Grafana 6.7.3 (`grafana/grafana:6.7.3`).

Any suggestions?

**Comment by ryantxu** [comment]
The signatures got mixed up at beta3... fixed either by running the nightly (7.1) and will be included in the 7.0 final

Try: `grafana/grafana:master`  until then

**Comment by hmitsch** [comment]
Works. 🎉  Thanks so much for addressing this so quickly!

**Comment by hmitsch** [comment]
Confirming that the sheets plugin also works on recently released grafana 7.0.0 (https://github.com/grafana/grafana/releases/tag/v7.0.0).
